api/src/app.ts

```diff
@@ -0,0 +1,36 @@
+/* eslint @typescript-eslint/no-var-requires: "off"
```

Comment:
Can changing `const` to `import` in the require lines below eliminate the need for this annotation?

Reply:
made change